File: SIMauricio/JAVA_TESTES/AULA02/Banco/ContaSalario.java
package Banco;

public class ContaSalario extends Conta {

    //Construtores da Classe Filha
    ContaSalario(){
    }
        
    ContaSalario(String agencia, String conta, float saldo){
        //Chama o Construtor da Classe Mãe
        super(agencia, conta, saldo); 
    }    

    public void lancarPgto(float valor){
        if (valor > 0.0f){
            this.saldo = this.saldo + valor;
        } else{
            System.out.println("Valor inválido!");
        }
    }
    
}

File: SIMauricio/PYTHON_TESTES/AULA02/Banco/ContaSalario.py
from Conta import Conta
class ContaSalario(Conta):
    def __init__(self, agencia, conta, saldo):
        super().__init__(agencia, conta, saldo)

    def lancarPgto(self, valor):
        if (valor >0.0):
            self.saldo = self.saldo + valor
        else:
            print('Valor inválido!')
